### DISCUSSION TITLE
How to administer users / admins

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#2885] [Self-Hosted Support]
Story:

First admin creates a website and invites a user as admin. 
Invitedadmin accepts invite.
Now i delete the website.

Invited Admin still exist. How can I mangage him? how can I delete him?
How can a  "server admin" see/update/remove all created/invited users on server? 
How can server admin see all websites created, also by other admins?

**Comment by ruslandoga** [comment]
👋 @jesperordrup 

I don't know of a way to do this through UI, but you might be able to do this by connecting to the postgres container and interacting with it directly. The relevant tables are `users`, `site_memberships`, and `invitations`.

For example, this is how you can list all users and their sites:

```console
$ cd hosting
$ docker compose exec plausible_db ash
# psql -U postgres -d plausible_db
```
```sql
select u.id, u.email, sm.role, s.domain from users u
left join site_memberships sm on sm.user_id = u.id
left join sites s on sm.site_id = s.id;
```
